onnxruntime/core/providers/webgpu/webgpu_context.cc

```diff
@@ -550,11 +564,11 @@ wgpu::Limits WebGpuContext::GetRequiredLimits(const wgpu::Adapter& adapter) cons
   required_limits.maxComputeWorkgroupsPerDimension = adapter_limits.maxComputeWorkgroupsPerDimension;
   required_limits.maxStorageBuffersPerShaderStage = adapter_limits.maxStorageBuffersPerShaderStage;
 
-  if (small_storage_buffer_binding_size_for_testing_) {
-    // No matter how small it is set, the minimum storage buffer binding size in WebGPU is 128 MB.
-    required_limits.maxStorageBufferBindingSize = 134217728;
-  } else {
+  if (max_storage_buffer_binding_size_ == 0) {
```

Comment:
Please also update the test case config https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/test/contrib_ops/matmul_4bits_test.cc#L551